components/calendar/src/cal/hijri.rs

```diff
@@ -178,12 +178,15 @@ impl Rules for AstronomicalSimulation {
 /// [^1]: See [calendrical_calculations::islamic::observational_islamic_from_fixed]
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
 #[non_exhaustive]
+#[cfg(feature = "unstable")] // TODO(before stabilisation): design API
```

Comment:
```suggestion
#[cfg(feature = "unstable")] // TODO(before stabilisation): should this be an enum? a struct with private fields?
```

I wanted to call out specifically that this is an important thing to consider, and that this aspect is in flux.